src/chrome-manifest.json

```diff
@@ -37,8 +37,7 @@
     "open_in_tab": true
   },
   "permissions": [
-    "*://*.toggl.com/*",
-    "*://toggl.com/*",
+    "*://track.toggl.com/*",
```

Comment:
I couldn't finish the login flow, I believe we still need access to `toggl.com` sans prefix

Look at these lines
https://github.com/toggl/toggl-button/blob/ce3fc5d34f317e3564ecbd82783bed2296498bab/webpack.config.js#L76
https://github.com/toggl/toggl-button/blob/16155d8560dfa2d9d6b1e550e6cc87fbebf989e8/src/scripts/components/LoginPage.tsx#L198

The extension needs access to `toggl.com/track/toggl-button-login/` to communicate with it and be informed when the login completes
https://github.com/toggl/toggl-button/blob/1f335016dc614a5289eac5b471bd9fbc8c93de2f/src/scripts/content/toggl.js#L37-L48

But this content script won't work unless it has access to the public-web page